jumpscale/tools/alerthandler/alerthandler.py

```diff
@@ -155,6 +157,7 @@ def alert_raise(
         data: dict = None,
         timestamp: float = None,
         traceback: dict = None,
+        send_alert: bool = True,
```

Comment:
not used anymore?

Reply:
this used from the new handler, cause I raise alert in case the alert can not be sent
or should we log instead?